### ISSUE TITLE
bug: TFOD Workflow Execution Rerun

### ISSUE THREAD

**Issue by inohmonton99** [onepanelio/core#790]
**Describe the bug**
non-deployed node appearing under node options when rerunning training
![Screenshot_1](https://user-images.githubusercontent.com/37641302/101978063-298b9180-3c8d-11eb-9139-1cda317ad246.png)

**`opctl` version**
```bash
$ opctl version
CLI version: v0.16.0-rc.2
Manifest version: v0.16.0-rc.2
API version: v0.16.0-rc.2
Web UI version: v0.16.0-rc.2
```

**`opctl init` command**
opctl init --provider eks ^
  --artifact-repository-provider s3 ^
  --gpu-device-plugins nvidia

**Kubernetes information**
 - Cloud provider: EKS
 - Kubernetes version: 1.18

**Machine information**
- OS: Windows 10
- Browser: Microsoft Edge

**Screenshots**

**Comment by rushtehrani** [comment]
Will be addressed by #763

**Comment by rushtehrani** [comment]
Closing this in favor of #763
